evaluate.sh

```diff
@@ -271,6 +271,12 @@ for fork in "$@"; do
     fi
   fi
 
+  # check if Java source file uses Unsafe
+  if grep -F "theUnsafe" -q ./src/main/java*/dev/morling/onebrc/CalculateAverage_$fork.java ; then
```

Comment:
This wouldn't work for those implementations where the fork name (i.e. GH account name) doesn't make for a valid Java class name (like in the case of `roman-s-m`). But that's alright, it doesn't matter that much in the large scheme of things.